.circle/data/ds005_outputs.txt

```diff
@@ -24,6 +24,7 @@ ds005/out/fmriprep/sub-01/anat/sub-01_T1w_space-MNI152NLin2009cAsym_class-WM_pro
 ds005/out/fmriprep/sub-01/anat/sub-01_T1w_space-MNI152NLin2009cAsym_dtissue.nii.gz
 ds005/out/fmriprep/sub-01/anat/sub-01_T1w_space-MNI152NLin2009cAsym_preproc.nii.gz
 ds005/out/fmriprep/sub-01/anat/sub-01_T1w_space-MNI152NLin2009cAsym_warp.h5
+ds005/out/fmriprep/sub-01/anat/sub-01_T1w_target-fsnative_affine.txt
```

Comment:
I thought the filename was `sub-01_T1w_preproc_target-fsnative_affine.txt` (missing `_preproc`)

Reply:
I did change it to make it more like `ds005/out/fmriprep/sub-01/anat/sub-01_T1w_space-MNI152NLin2009cAsym_warp.h5`, which is a transform from `T1w` space to `MNI152NLin2009cAsym`. Speaking of which, @chrisfilo, reading the BEP, should that be a `target-MNI...` instead of `space-MNI...`?

Reply:
Correct.